packages/dev/core/src/assetContainer.ts

```diff
@@ -434,7 +443,7 @@ export class AssetContainer extends AbstractScene {
             } else {
                 // Mesh or TransformNode
                 let canInstance = true;
-                if (node.getClassName() === "TransformNode" || (node as Mesh).skeleton || (node as Mesh).getTotalVertices() === 0) {
+                if (node.getClassName() === "TransformNode" || (node as Mesh).skeleton || (node.getClassName() === "Mesh" && (node as Mesh).getTotalVertices() === 0)) {

```

Comment:
I think we should also disable instancing if node === instance of `Node`

Reply:
Oh, easier, we could just check if the className !== Mesh || InstancedMesh as that's the only class we can instance